### DISCUSSION TITLE
Show resolution in the Image Gallery

### DISCUSSION THREAD

**Discussion by dalogue1** [monkeymademe/CamUI#71] [General]
The Image Gallery currently shows the date the picture was taken.  Is there a possibility of showing the image resolution as well.

![image](https://github.com/user-attachments/assets/a7f7d8c9-f7d5-40ab-b238-3945e23e2500)

**Comment by monkeymademe** [comment]
Reminder please post feature requests or bugs in the Issues tab ;-)

Don't need to for this.. I will take care of it later its an easy add

**Comment by monkeymademe** [comment]
If you pull from main its there
<img width="365" alt="image" src="https://github.com/user-attachments/assets/07acb87c-d059-461c-8fe9-7666e09e25bf">
